### ISSUE TITLE
Rails API authentication on POST routes etc

### ISSUE THREAD

**Issue by jonathanpeterwu** [jonathanpeterwu/tab#6]
Hey @katherineimogene no rush on this but was wondering if you knew how to do authenticated POST requests to the app outside of a rails form. Do you guys do that at work?

Resources
[Building Restful APIs](https://www.airpair.com/ruby-on-rails/posts/building-a-restful-api-in-a-rails-application)
[Minimal Authentication for APIs](http://resistor.io/blog/2013/08/07/mimimal-api-authentication-on-rails/)

**Comment by jonathanpeterwu** [comment]
@katherineimogene 

Hey just a thought, not sure if you've ever dived into the rails-api gem.

https://github.com/rails-api/rails-api

**Comment by katherineimogene** [comment]
@jonathanpeterwu you read my mind. 
I think I'd like to pivot a little and make the whole backend behave like an API. It'll make both our lives easier. More deets soon!

**Comment by jonathanpeterwu** [comment]
I agree! Here's some rails-api GEM tutorials if you wanna check them out

Rails API Tutorial
https://www.amberbit.com/blog/2014/2/19/building-and-documenting-api-in-rails/
http://railscasts.com/episodes/348-the-rails-api-gem
https://github.com/rails-api/rails-api
